app/view_models/profile_index.rb

```diff
@@ -16,13 +16,21 @@ def personal_key_partial
   end
 
   def password_reset_partial
-    if current_user.password_reset_profile.present?
+    if current_user.decorate.password_reset_profile.present?
```

Comment:
if `decorated_user` delegates to `current_user` (which I think it should, if it doesn't already), we could *just* pass `decorated_user` to the view model.